### ISSUE TITLE
Plastic Box becomes a different item after placing and picking it up

### ISSUE THREAD

**Issue by oversword** [obl3pplifp/BlS-issue-tracker#302]
Whaat?

**Comment by pl608** [comment]
probably either a conflict with the name or someone set what it would drop once digged(you can do that you know)

**Comment by oversword** [comment]
I think it's to do with the fact it can be dyed any of the 256 colours, maybe whatever mod is doing that is messing it up with metadata?

**Comment by pl608** [comment]
yeah that probably it, also what is it dropping? undyed plasticbox?

**Comment by oversword** [comment]
I just checked, it seems to drop the same node name that you craft, it just doesn't stack, must be metadata

Maybe we just need to clear the metadata when you pick it up and it's undyed (I assume the metadata is for dyes), probably easier than adding metadata when it's crafted

**Comment by oversword** [comment]
Hmm I wonder how the dying works with metadata...

**Comment by pl608** [comment]
or make it drop both the and the plastic. Are all dyed products a diffrent object or does it just change the color?

**Comment by pl608** [comment]
also which file is this in?

**Comment by oversword** [comment]
Looks like we might need to make these changes to unifieddyes itself, maybe try updating the version first and see if that fixes it

**Comment by oversword** [comment]
> also which file is this in?

You're going to have to figure out that kind of thing for yourself xD

Look at the node name, the first part should be the mod, then just guess from there. If you still can't find it do a search on the mods folder

**Comment by pl608** [comment]
ok will do
